tools/authors/harvest.php

```diff
@@ -196,37 +198,30 @@
                         byear, bmonth, bday,
                         dyear, dmonth, dday,
                         bcomments, dcomments, enabled)
-                VALUES ('%s', '%s', $date_fields_str '', '', 'no')
-            ", mysqli_real_escape_string(DPDatabase::get_connection(), $last_name),
-                mysqli_real_escape_string(DPDatabase::get_connection(), $other_names));
+                VALUES ('%s', '%s', %s '', '', 'no')
```

Comment:
Leave `$date_fields_str` inside the string since its contents are already validated prior.